src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -28,6 +28,7 @@ import AllFixedLogo from "./images/dashboard-all-fixed.svg";
 import { FeatureFlagsEnabled } from "../../../../../functions/server/featureFlags";
 import { filterExposures } from "./filterExposures";
 import { SubscriberBreach } from "../../../../../../utils/subscriberBreaches";
+export const ONEREP_DATA_BROKER_COUNT = 190;
```

Comment:
Not sure if we ever figured out a solution for hardcoding the broker count everywhere. I thought we had an ENV var for this, but maybe we removed it.

```sh
git grep -Pn "\b190\b" -- ":(exclude)*.svg" ":(exclude)*.json" ":(exclude)*.ftl" | cat

src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx:31:export const ONEREP_DATA_BROKER_COUNT = 190;
src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/automatic-remove/page.tsx:19:  const dataBrokerCount = 190;
src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/start-free-scan/page.tsx:20:  const dataBrokerCount = 190;
src/app/(proper_react)/redesign/(authenticated)/user/welcome/Onboarding.stories.tsx:21:      dataBrokerCount={190}
src/app/components/client/ExposuresFilterExplainer.tsx:30:          data_broker_sites_total_num: 190,
src/app/components/client/ExposuresFilterExplainer.tsx:61:          data_broker_sites_total_num: 190,
src/app/functions/server/onerep.ts:80:export const ONEREP_DATA_BROKER_COUNT = 190;
```

Reply:
We can probably import the `const` from src/app/functions/server/onerep.ts file up on L23. (although that still leaves the 6 other hardcoded values; but that might be for a future PR).

Reply:
Might be a future PR.. I'm not sure if there's a consensus yet, but I think one approach is to store it as a value in the db. There seems to be a few issues with storing it as env var:
1) changing the number on the fly is hard (especially for stage/prod)
2) some hydration error for the views when attempting to parse out the env var
 (see `src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/start-free-scan/page.tsx`)